packages/framework/src/Resources/scripts/admin/product.js

```diff
@@ -19,11 +19,6 @@
         Shopsys.product.toggleIsUsingAlternateAvailability($outOfStockActionSelection.val() === Shopsys.constant('\\Shopsys\\FrameworkBundle\\Model\\Product\\Product::OUT_OF_STOCK_ACTION_SET_ALTERNATE_AVAILABILITY'));
 
         Shopsys.product.initializeSideNavigation();
-
-        $('#js-close-without-saving').on('click', function () {
```

Comment:
I'm not sure whether it is a good idea to remove such functionality, the implementator can rely on it. If you need to remove it, please create upgrade instructions

Reply:
I have added upgrade note

Reply:
content is correct, but wrong upgrade.md, please move it to `upgrade-v8.0.0.md`

Reply:
Oh you are right!